src/components/pages/devices/flyouts/deviceNew/deviceNew.js

```diff
@@ -129,16 +129,24 @@ const ProvisionedDevice = ({ device, t }) => {
 export class DeviceNew extends LinkedComponent {
   constructor(props) {
     super(props);
+
+    const { t } = this.props;
```

Comment:
`t` doesn't need to be in the constructor. The constructor is only fired once but the `t` function needs to fire if the props change.

Reply:
Huh? t is used in all the validation messages... that are set up within the constructor. See lines 162-193.